clients/feeder/feeder.go

```diff
@@ -222,6 +222,9 @@ func (c *Client) get(ctx context.Context, queryURL string) (io.ReadCloser, error
 					return res.Body, nil
 				} else {
 					err = errors.New(res.Status)
+					if err.Error() == "400 Bad Request" {
```

Comment:
Can we check the `res.Status` instead of matching strings?

Reply:
The added check here will be removed and just set the error to res.Status like we're doing now as the new endpoint is available on mainnet.